.circleci/config.yml

```diff
@@ -514,8 +563,7 @@ jobs:
 
   # Playground job. Re-run with ssh enabled to play with Circle's machine image.
   playground:
-    machine:
-      image: ubuntu-2204:2022.04.1
+    executor: pact-executor
```

Comment:
I think the other image makes more sense as a default

Reply:
Whoops! Good call. I believe that I was using it here for a quick way to debug.

Reply:
I figured :)